src/core/algorithms/gwo.cpp

```diff
@@ -0,0 +1,61 @@
+#include <stdexcept>
+#include <core/algorithms/gwo.h>
+
+namespace dnn_opt
+{
+namespace core
+{
+namespace algorithms
+{
+
+void gwo::optimize()
+{
+
+}
+
+solution* gwo::get_best()
+{
+
+}
+
+void gwo::set_params(std::vector<float> &params)
+{
+
+}
+
+void gwo::reset()
+{
+
+}
+
+void gwo::update_Elite()
+{
+  for(int i =0;i<=solution.size();i++){
+    solution* current = get_solutions()->get(i);
+    if (current->is_better_than(_alpha,is_maximization())) {
```

Comment:
Code standards suggeset to leave an empty space after the colon, the same issue happens in several lines